playbooks/byo/generate_prometheus.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I believe this location is fine, but the naming to be consistent should use dashes instead of underscore. You may even wish to name it 'openshift-prometheus' to follow the logging and metrics pattern